py_zipkin/zipkin.py

```diff
@@ -244,7 +245,6 @@ def start(self):
 
         self.start_timestamp = time.time()
 
-        # Set up logging if this is the root span
         if self.perform_logging:
```

Comment:
Not _quite_. The case where we want to perform logging but don't want to report timestamp/duration is when zipkin_attrs are passed in (implying that this span is a continuation of another span, which implies that the client part of this span will be logged elsewhere and should contain timestamp/duration) and this span isn't independently sampled.